### PR TITLE
[DOCS-9708] Add Action logic for RUM

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

- Adds the Action debouncing logic to the following pages:
  - Browser
  - Android
  - iOS

I also noticed that some of the guide titles were incorrectly capitalized so I updated them in this PR as well.

### Merge instructions

Merge readiness:
- [ ] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/real_user_monitoring/browser/tracking_user_actions.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ The RUM Browser SDK automatically tracks clicks. A click action is created if **
 * The click does not lead to a new page being loaded, in which case, the Datadog Browser SDK generates another RUM View event.
 * A name can be computed for the action. See [Declaring a name for click actions](#declare-a-name-for-click-actions) for details.
 
+**Note**: When an action is being tracked, other actions within the next 100 ms do not get sent, unless they are [custom actions][5].
```
Comment (brett0000FF) [inline content/en/real_user_monitoring/browser/tracking_user_actions.md:44]:
```suggestion
**Note**: When an action is being tracked, other actions within the next `100 ms` do not get sent, unless they are [custom actions][5].
```
Just matching the monospace formatting in `content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/android.md` for consistency.

**Review by brett0000FF — APPROVED** [review]
Approved with one small suggestion! 🚀

File: content/en/real_user_monitoring/browser/tracking_user_actions.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/browser/tracking_user_actions.md]: good suggestion - thanks!

**Review by xgouchet — APPROVED** [review]

**Pull request merged**